desktop/src/play_scene_slc.nim

```diff
@@ -64,12 +74,19 @@ proc enterPlayScene(scene: Scene, game: Game, tick:int) =
 
 proc updatePlayScene(scene: Scene, game: Game, tick:int) =
   # called on game update proc
+
+  # update play scene HUD here
+  game.drawHUD(tick)
```

Comment:
this is **wrong** for two reasons.
 1. :fire: do not ever do rendering code in update it can cause buggy problems and even crashes to the application
 2. you do not need to call the same procedure in two places. once in the render is enough 😉

Reply:
makes sense. Addressed by [7082803](https://github.com/RichardMarks/coin-collector-mobile/pull/34/commits/70828039c1f66d5f96feefe087a34891e36e735c)